docs/intro/tutorial-2.rst

```diff
@@ -0,0 +1,17 @@
+Tutorial: Running Python code using Batavia from the command line
+=================================================================
+
+Batavia includes a simple command-line script that you can use to run your
+Python code. To use this tool you need to have followed the instructions from
+:doc:`tutorial-0`.
+
+You can now run Python code from a code from the command line as follows:
+
+.. code-block:: bash
+
+    node run_in_batavia.js /path/to/python_file.py
```

Comment:
The standard way to [call scripts in npm packages](https://docs.npmjs.com/cli/run-script) is `npm run {script_name}`. 

Could we add a script entry to the `package.json` so the command is something like `npm run python /path/to/python/file`?

Reply:
Certainly, will add that.